### PR TITLE
Feature/student model

### PR DESCRIPTION
**Pull request by magdalenagola**

### CODE REVIEW

File: com/codecool/java/dao/DbLoginDAO.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    @Override
+    public User loadOne(String login, String password) throws SQLException {
+        Connection c = pool.getConnection();
+        PreparedStatement ps = c.prepareStatement("SELECT * FROM USER" +
+                "WHERE email = ? AND password = ?;");
+        ps.setString(1, login);
+        ps.setString(2, password);
+        ResultSet rs = ps.executeQuery();
+        UserFactory uf = new UserFactory();
```
Comment (e-victoria) [inline com/codecool/java/dao/DbLoginDAO.java]: Please, change variable to more understandable name(userFactory, for example)

File: com/codecool/java/model/Student.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        this.password = password;
+        this.name = name;
+        this.surname = surname;
+        this.isActive = isActive;
+    }
+
+    @Override
+    public void start(){
+        StudentController sc = new StudentController();
+        sc.run();
```
Comment (e-victoria) [inline com/codecool/java/model/Student.java]: The same here. studentController would be a better name

File: com/codecool/java/dao/DbLoginDAO.java (hunk 1 above)
Comment (e-victoria) [reply com/codecool/java/dao/DbLoginDAO.java]: The same with other DbDAOs

File: com/codecool/java/model/Student.java (hunk 2 above)
Comment (e-victoria) [reply com/codecool/java/model/Student.java]: The same with other DbDAOs

File: com/codecool/java/dao/DbLoginDAO.java (hunk 1 above)
Comment (magdalenagola) [reply com/codecool/java/dao/DbLoginDAO.java]: Both are refactored now

**Pull request merged**